src/java/org/apache/cassandra/schema/ColumnMetadata.java

```diff
@@ -609,6 +689,9 @@ public void serialize(ColumnMetadata t, DataOutputPlus out, Version version) thr
             out.writeBoolean(t.mask != null);
             if (t.mask != null)
                 ColumnMask.serializer.serialize(t.mask, out, version);
+            out.writeBoolean(!t.columnConstraints.isEmpty());
```

Comment:
`t.columnConstraints` could be `null`. 

Can you remove the nullness of the field? The default is no-op constraint, instead of `null`.